src/core/shape_inference/include/concat_shape_inference.hpp

```diff
@@ -4,9 +4,9 @@
 
 #pragma once
 
-#include "openvino/core/validation_util.hpp"
 #include "openvino/op/concat.hpp"
 #include "utils.hpp"
+#include "validation_util.hpp"
```

Comment:
If current include path will be kept then this change will be not required.

Reply:
But then there'd be conflict between targeted include directories till legacy validation_util.hpp is removed.